$NetBSD$

Install config to example directory; will be installed
with pkgsrc config framework.

--- pwm/Makefile.orig	2011-10-29 18:19:21.000000000 +0000
+++ pwm/Makefile
@@ -49,7 +49,7 @@ include $(TOPDIR)/build/rules.mk
 ######################################
 
 _install: executable_install
-	$(INSTALLDIR) $(DESTDIR)$(PWM_ETCDIR)
+	$(INSTALLDIR) $(DESTDIR)$(EXAMPLEDIR)
 	for i in $(ETC); do \
-		$(INSTALL) -m $(DATA_MODE) $$i $(DESTDIR)$(PWM_ETCDIR); \
+		$(INSTALL) -m $(DATA_MODE) $$i $(DESTDIR)$(EXAMPLEDIR); \
 	done
